getting-started/configuring.md

```diff
@@ -63,7 +63,7 @@ and append the suggestions to the end of your `postgresql.conf` like so:
 $ timescaledb-tune --quiet --yes --dry-run >> /path/to/postgresql.conf
 ```
 
-## Further configuration / manual tuning [](further-config)
+## Postgres configuration and tuning [](postgres-config)
```

Comment:
@mfreed - these are my current headings (below for TimescaleDB too). Let me know what you think.